src/aura/BGE_EntryForm/BGE_EntryForm.cmp

```diff
@@ -35,18 +35,133 @@
+                                <lightning:formattedText value="{!$Label.c.bgeFormSelectedPayment}"/>
+                                <aura:set attribute="else">
+                                    <lightning:button
+                                            value="{!donation}"
+                                            label="{!$Label.c.bgeFormUpdatePayment}"
+                                            onclick="{!c.handleDonationSelection}"
+                                            variant="base"/>
+                                </aura:set>
+                            </aura:if>
+                        </aura:set>
+
+                        <lightning:layout class="slds-p-horizontal_medium" multipleRows="true">
+                            <lightning:layoutItem size="4" smallDeviceSize="6">
+                                <lightning:formattedText value="{!v.labels.opportunityName + ': '}"/>
+                            </lightning:layoutItem>
+                            <lightning:layoutItem size="8" smallDeviceSize="6">
+                                <lightning:formattedText value="{!donation.npe01__Opportunity__r.Name}"/>
+                            </lightning:layoutItem>
+                            <lightning:layoutItem size="4" smallDeviceSize="6">
+                                <lightning:formattedText value="{!v.labels.paymentAmount + ': '}"/>
+                            </lightning:layoutItem>
+                            <lightning:layoutItem size="8" smallDeviceSize="6">
+                                <ui:outputCurrency value="{!donation.npe01__Payment_Amount__c}"/>
+                            </lightning:layoutItem>
+                            <lightning:layoutItem size="4" smallDeviceSize="6">
+                                <lightning:formattedText value="{!v.labels.paymentScheduledDate + ': '}"/>
+                            </lightning:layoutItem>
+                            <lightning:layoutItem size="8" smallDeviceSize="6">
+                                <lightning:formattedText value="{!donation.npe01__Scheduled_Date__c}"/>
+                            </lightning:layoutItem>
+                        </lightning:layout>
+                    </lightning:card>
+                </lightning:layoutItem>
+            </aura:iteration>
+            <aura:iteration var="donation" items="{!v.openOpportunities}">
+                <lightning:layoutItem largeDeviceSize="4" mediumDeviceSize="6" smallDeviceSize="12" size="12" padding="around-small">
+                    <lightning:card title="{!donation.Name}" iconName="standard:opportunity"
+                                    class="slds-card_boundary">
+                        <aura:set attribute="footer">
+                            <aura:if isTrue="{!equals(donation.Id, v.selectedDonation.Id)}">
+                                <!--<aura:if isTrue="{!donation.applyPayment}">-->
```

Comment:
Parked for when we add in the Apply New Payment option. We had it exposed for a while so we could see what it looks like when going back and forth with Shannon, but commented it out because it's not actually part of the scope of this ticket.